src/test/java/com/dawn/jat/illuminati/post/controller/PostControllerTest.java

```diff
@@ -62,34 +58,24 @@ void findPostAll_whenNoRecord() {
 
     @Test
     void findPostAll_whenRecord() {
-        Mockito.when(postService.findAll()).thenReturn(Arrays.asList(postEntity1));
-        assertThat(postController.findAll().size(), is(1));
+        Mockito.when(postService.findAll()).thenReturn(Arrays.asList(postEntity1, postEntity2));
```

Comment:
separate this into 2 parts 
List<String> mockPostEntities = Arrays.asList(postEntity1, postEntity2);
Mockito.when(postService.findAll()).thenReturn(mockPostEntities)

Reply:
Updated